src/utils/csvToJson.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-useless-return */
+const { Transform } = require('stream');
+
+const { parseString } = require('fast-csv');
+
+function createCsvToJson() {
+  const transform = (chunk, encoding, callback) => {
+    let result = '';
+
+    const stringProducts = chunk.toString('utf8');
+    let stringProductsWithoutRemnant = stringProducts.slice(
+      0,
+      stringProducts.lastIndexOf('\n'),
+    );
+
+    if (this.isNotFirst) {
+      if (this.remnant) {
+        stringProductsWithoutRemnant = `${this.remnant}${stringProductsWithoutRemnant}`;
+      }
+
+      result += ',\n';
+    } else {
+      this.isNotFirst = true;
+      [this.columnsHeaders] = stringProductsWithoutRemnant.split('\n', 1);
+      stringProductsWithoutRemnant = stringProductsWithoutRemnant.slice(
+        stringProductsWithoutRemnant.indexOf('\n') + 1,
+      );
+      this.rowCount = 0;
+
+      result += '[';
+    }
+
+    this.remnant = stringProducts.slice(stringProducts.lastIndexOf('\n') + 1);
+
+    const addAllColumnsInProduct = arrayValuesProduct => {
+      const arrayNamesColumns = this.columnsHeaders.split(',');
+
+      const product = {};
+      arrayValuesProduct.forEach((valueProduct, index) => {
+        product[arrayNamesColumns[index]] = valueProduct;
+      });
```

Comment:
That's why `Array.map()` was invented.

Reply:
No, because I create need `Object`. If I`ll you `Array.map()` it will return array of objects instead of one Object with many types like this
![Screenshot from 2020-12-02 22-20-11](https://user-images.githubusercontent.com/56438694/100926917-cd867780-34ec-11eb-8ace-3b44b7e8a8ed.png)